app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
With the method definition I would never look for it in the view. The conditional, at least, is all in one place.

I'm skeptical of putting view logic in the model.